### PR TITLE
=pro Update README to include intro and fix links (#319)

### PR DESCRIPTION
**Pull request by jonas**
This might need some touch ups from the core team but this should bring it closer.

### CODE REVIEW

**Comment by ktoso** [comment]
LGTM, please have a look @akka/akka-http-team if wording sounds good and let's merge :)

File: README.md [hunk 1]
```diff
@@ -1,60 +1,73 @@
...
+[github-issues]:       https://github.com/akka/akka-http/issues
+[scaladex-badge]:      https://index.scala-lang.org/count.svg?q=dependencies:akka/akka-http*&subject=scaladex:&color=blue&style=flat-square
+[scaladex-projects]:   https://index.scala-lang.org/search?q=dependencies:akka/akka-http*
 
 Contributing
 ------------
 Contributions are *very* welcome!
 
-If you see an issue that you'd like to see fixed, the best way to make it happen is to help out by submitting a PullRequest implementing it.
+If you see an issue that you'd like to see fixed, the best way to make it happen is to help out by submitting a pull request.
```
Comment (jonas) [inline README.md]: Here could be a link saying:

For ideas of where to contributing [these tickets](https://github.com/akka/akka-http/issues?q=is%3Aissue+is%3Aopen+label%3Acommunity) are a good starting point.

Comment (ktoso) [inline README.md]: yes good idea, please add 👍

**Comment by ktoso** [comment]
LGTM

**Comment by ktoso** [comment]
I'll add the note about tickets to start

**Comment by ktoso** [comment]
Thanks for the PR!

**Pull request merged**